components/admin_console/data_retention_settings/data_retention_settings.tsx

```diff
@@ -99,9 +103,23 @@ export default class DataRetentionSettings extends React.PureComponent<Props, St
                 className: 'actionIcon',
             },
         ];
+        if (this.includeBoards) {
+            columns.splice(columns.length - 1, 0,
```

Comment:
nit: This splice is a bit hard to understand in the intentions. I would just replace that with a `columns.push` here, and move the last entry of for the `actions` field, in another `column.push` at the end (right after the if). That way is easier to understand the "flow" of the data.